.github/workflows/build.yml

```diff
@@ -1,15 +1,7 @@
 ---
 name: Build
 
-on:
-  push:
-    paths-ignore:
-      - 'docs/**'
-      - '*.md'
-  pull_request:
-    paths-ignore:
-      - 'docs/**'
-      - '*.md'
```

Comment:
Do we have to run the `build` workflow on those paths? Asking since only the `check-formatting` workflow is required for merging.

Reply:
I must have gotten them mixed up when I wrote the above description. `build` is actually required

![image](https://user-images.githubusercontent.com/1629196/90870912-94415600-e39a-11ea-90fa-aaab07b65895.png)

Reply:
Interesting. On other PRs (e.g. #1883), the `check-formatting` workflow has the "Required" badge:
![image](https://user-images.githubusercontent.com/35232120/90871144-e71b0d80-e39a-11ea-9754-ee2ab9409c01.png)

Reply:
Oh right, the workflow is called `Check formatting` and the job inside the workflow is `build`, that's probably why. So you were right.

Reply:
That means we should be able to keep the ignores in the `build.yml` file, right? Or am I mixing something up now?

Reply:
just applied your suggestion, thanks